### DISCUSSION TITLE
DA5413EC-FAC5-438E-8190-ADB1066CAAD9

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#24] [General]
# Kepler-1513

Kepler-1513, also known as KOI-3678 and Gaia-TROI-127, is a K-type main sequence star about 1,147 light-years away in the constellation Lyra. In 2022, studies of the transit-timing variations of its one known planet suggested the existence of a Moon-to-Mars-mass satellite, making it the only the third strong candidate exomoon and the smallest one detected so far (the first two exomoons at Kepler-1625 and Kepler-1708 were Neptune-sized).

https://celestia.mobi/addon?item=DA5413EC-FAC5-438E-8190-ADB1066CAAD9

**Comment by exocritic5457** [comment]
pretty nice, do wish the moon could cast shadows though

**Reply by TheLostProbe** [reply]
it should do that if you have eclipse shadows enabled

**Reply by exocritic5457** [reply]
Regardless of the setting and the moon lining up in the right area, no shadow shows up

**Reply by TheLostProbe** [reply]
that's really odd...

**Reply by kmjordan122313-lgtm** [reply]
it is still a good addon.

**Comment by exocritic5457** [comment]
Unfortunately it seems it was foreshadowed, and the moon casts no shadow because it doesn't actually exist.

**Reply by DaveBowman2001** [reply]
The Eclipse Finder tool shows that the (now disproven) moon *do* in fact cast shadows on its host planet frequently. However, it is too small to effectively block their star completely (see image below), making the shadows of its solar eclipses negligible. Total lunar eclipses for this moon are a common event in this addon though as a result:
![Screenshot_2025-04-30-23-36-09-124_space celestia mobilecelestia](https://github.com/user-attachments/assets/f2387c16-d891-4f6f-9ea8-44ff31733c48)

**Comment by AU-Microscoppi-b** [comment]
The texture is glitched with the generic gas giant texture on mobile

**Reply by DaveBowman2001** [reply]
One of the add-on devs on Discord pinpointed it to be caused by an outdated code in the SSC file, which is written two years before we knew [Kepler-1513 is indeed a binary star system](https://academic.oup.com/mnras/article/527/2/3183/7379625) (_Michel, 2024_)

We've sent a patch update for this add-on a few minutes ago (as of this writing), this should be able to fix the visual defects reported. Thanks

**Reply by AU-Microscoppi-b** [reply]
Also ima warn you rn I’m looking into every exoplanet addon to see if they have problems. Don’t rush, take your time I’m just someone reporting problems (all problems are on mobile)

**Comment by kmjordan122313-lgtm** [comment]
good addon, i'm interested at exomoon candidates.
